HaruhiGekidouTests/SourceTests.cs

```diff
@@ -44,6 +47,52 @@ public class SourceTests
             "Tutorial_006"
         ];
 
+        public static string[] _AdvPartScriptFiles()
+        {
+            if (Directory.Exists("./input/AdvPartScript/"))
```

Comment:
I realize I didn't do this in Choku, but for these directories I'd generally recommend using `Path.Combine` as it's very portable. e.g.:

```suggestion
            if (Directory.Exists(Path.Combine("input", "AdvPartScript"))
```